### PR TITLE
feat: Update asserts to use new winter23 Assert class - Platform Cache Recipes

### PR DESCRIPTION
**Pull request by msrivastav13**
### What does this PR do?

Update asserts to use new winter23 Assert class. The new Assert class is really elegant compared to the older System methods.

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls [hunk 1]
```diff
@@ -105,7 +105,7 @@ private class PlatformCacheRecipes_Tests {
         Test.startTest();
         PlatformCacheRecipes.removeKeyFromSessionCache('Account');
         Test.stopTest();
-        System.assert(
+        System.Assert.isTrue(
```
Comment (pozil) [inline force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls:108]: Change to `isFalse`

File: force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls [hunk 2]
```diff
@@ -217,7 +217,7 @@ private class PlatformCacheRecipes_Tests {
         Test.startTest();
         PlatformCacheRecipes.removeKeyFromOrgCache('Account');
         Test.stopTest();
-        System.assert(
+        System.Assert.isTrue(
```
Comment (pozil) [inline force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls:220]: Change to `isFalse`

**Review by pozil — CHANGES_REQUESTED** [review]
Found some minor issues.

File: force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls (hunk 2 above)
Comment (msrivastav13) [reply force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls]: Good catch. Fixed!

File: force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls (hunk 1 above)
Comment (msrivastav13) [reply force-app/tests/Platform Cache Recipes/PlatformCacheRecipes_Tests.cls]: Good catch. Fixed!

**Review by pozil — APPROVED** [review]
Thanks for the update.

**Pull request merged**